enterprise/api/v1/serializers.py

```diff
@@ -673,6 +689,13 @@ def validate(self, data):  # pylint: disable=arguments-differ
         tpa_user_id = data.get('tpa_user_id')
         user_email = data.get('user_email')
         if not lms_user_id and not tpa_user_id and not user_email:
+            error_message = ('[Enterprise API] The fields: lms_user_id={lms_user_id},'
```

Comment:
this case is really more that the user did not provide any of the three fields needed to process the request and at least one should be provided.

Reply:
Added additional context.